Sources/TuistDependencies/DependenciesController.swift

```diff
@@ -114,24 +119,34 @@ public final class DependenciesController: DependenciesControlling {
             throw DependenciesControllerError.noPlatforms
         }
 
+        var dependenciesGraph: DependenciesGraph?
```

Comment:
Maybe it's easier to make it non-nillable, so that for each package manager we just need to merge the content with the output of the package manager, without any special handling of the first one.
Then, the last `if` just check whether it's empty instead of nil, which would also be a bit more robust, as now we might create an empty json in case dependenciesGraph is not nil but empty